UOP1_Project/Assets/Scripts/Pool/Example/PoolTester.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PoolTester : MonoBehaviour
+{
+	[SerializeField]
+	private ParticlePool _pool = default;
+
+	private IEnumerator Start()
+	{
+		List<PoolableParticle> particles = _pool.Request(10) as List<PoolableParticle>;
+		foreach(PoolableParticle particle in particles)
+		{
+			particle.transform.position = Random.insideUnitSphere * 5f;
+			particle.Play();
+		}
+		yield return new WaitForSecondsRealtime(5f);
```

Comment:
Why using `WaitForSecondsRealtime` here? Since you're dealing with ParticleSystems, if you scale time they will go slowly, so you should used scaled time here as well (`WaitForSeconds`).

Reply:
Yes, good point. My example was less about the specific particle system implementation and more to showcase how to use the pool. I have to admit I'm no particle system expert :) But I should have caught that.